docs/kb/semgrep-appsec-platform/saml-authentication-method-match.md

```diff
@@ -0,0 +1,26 @@
+---
+description: If needed, check the box to enable non-password authentication mechanisms on Semgrep AppSec Platform.
+tags:
+  - Semgrep AppSec Platform
+  - SSO
+  - SAML
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# SAML SSO Error: Authentication method doesn't match requested
+
+When logging in to Semgrep using SAML single-sign on (SSO), you may see an error `Authentication method doesn't match requested`. The error might look something like this:
```

Comment:
```suggestion
When logging in to Semgrep using SAML single-sign on (SSO), you may see the error `Authentication method doesn't match requested`:
```

Reply:
I'm pretty ambivalent, we can keep or remove. I'd just change the "an" to a "the"

Reply:
yeah I liked that part of the change. I think I'm just going to commit your version for now